packages/asset-swapper/CHANGELOG.json

```diff
@@ -1,14 +1,23 @@
 [
     {
-<<<<<<< development
-        "version": "16.55.1",
+        "version": "16.57.2",
+        "changes": [
+            {
+                "note": "Added cvxFXS/FXS curve pool on mainnet",
+                "pr": 465
+            }
+        ]
+    },
+    {
+        "version": "16.57.1",
```

Comment:
Since this version hasn't been published yet (indicated by lack of `timestamp`) you can feel free to add your change into the previous `changes` section.